### ISSUE TITLE
Spaces getting stripped from parsed files.

### ISSUE THREAD

**Issue by jzohrab** [sakolkar/openepub#2]
Hey there!

In issue https://github.com/jzohrab/lute-v3/issues/344, there's an image showing how spaces are getting stripped during epub import.  e.g., Here's a screenshot of the source epub:

> <img width="443" alt="image" src="https://github.com/sakolkar/openepub/assets/1637133/b299fe3d-ceff-4674-9c9f-63703b2eae81">

Here's how it appears in Lute:

> <img width="866" alt="image" src="https://github.com/sakolkar/openepub/assets/1637133/2f8f7a80-1d5d-4d77-be62-c3d8202c6728">

Here's the actual source in the zipped epub html:

```
<p class="Clanek1"><span lang="CS">Odpovědi na tyto a mnohé další otázky hledá v novém senzačním životopise
nazvaném </span><span lang="CS" class="calibre8">Život a lži Albuse Brumbála</span><span lang="CS"> Rita Holoubková. Exkluzivní rozhovor s ní přináší Betty Braithwaiteová
na straně 13.</span></p>
```

Ideally, the above snippet would be extracted as:

```
Odpovědi na tyto a mnohé další otázky hledá v novém senzačním životopise nazvaném Život a lži Albuse Brumbála Rita Holoubková. Exkluzivní rozhovor s ní přináší Betty Braithwaiteová na straně 13.
```

Does this seem like it's possible?  Cheers!

**Comment by sakolkar** [comment]
@jzohrab An initial fix is implemented so this is definitely fixable. I need to craft some other scenarios around this, however, before I call it complete. Sometimes the spaces could be between spans and not within and those should also be preserved.

**Comment by jzohrab** [comment]
Cheers thanks for taking the time!  Hard to say what data books will give us, isn't it.

**Comment by sakolkar** [comment]
You're welcome! I added some unit tests for full epub text extractions to ensure those are still working correctly.

This is now fixed in 0.0.8 and which is also now available on PyPI
